MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -188,6 +190,9 @@ + (void)notifyWithUserConfirmation:(MSUserConfirmation)userConfirmation {
     // Get error attachments.
     if ([crashes delegateImplementsAttachmentCallback]) {
       attachments = [crashes.delegate attachmentsWithCrashes:crashes forErrorReport:report];
+      if (attachments.count > kMaxAttachmentsPerCrashReport) {
```

Comment:
Call [attachement isValid] before [line 206](https://github.com/Microsoft/mobile-center-sdk-ios/pull/575/files#diff-434ca6481b9819743235b3b829b57787R206), only process attachment log if it is valid. Otherwise, please log the error.
This is the error message in Android.
`MobileCenterLog.error(LOG_TAG, "Not all required fields are present in ErrorAttachmentLog.");`